frontend/src/lib/candidate/components/PasswordSetComponent/PasswordSetComponent.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+  import {t} from '$lib/i18n';
+  import {Button} from '$lib/components/button';
+  import PasswordField from '../PasswordField/PasswordField.svelte';
+  import {getRoute, Route} from '$lib/utils/navigation';
+  import PasswordValidator from '../passwordValidator/PasswordValidator.svelte';
+  import {createEventDispatcher} from 'svelte';
+  export let password1 = '';
+  export let password2 = '';
+  export let autocomplete1 = '';
```

Comment:
The autocomplete values could have a default value, whichever is more common with this component